### DISCUSSION TITLE
Both WSL & WSL2 are required？

### DISCUSSION THREAD

**Discussion by maicss** [podman-desktop/podman-desktop#5092] [General]
![image](https://github.com/containers/podman-desktop/assets/9784767/6a5c8e59-46bc-45f9-9041-46b46d2b1dbc)
![image](https://github.com/containers/podman-desktop/assets/9784767/28dd0f05-b8e9-4aa1-981d-0153c9688339)

**Comment by jeffmaury** [comment]
WSL version 2 is required to run Podman on Windows.

**Comment by maicss** [comment]
As the screenshot shows, I have installed WSL version 2 on my windows, why stop me from installing or other operations?

**Reply by jeffmaury** [reply]
There is an issue that has been fixed a few days ago regarding getting WSL version in certains environments, can you check with the nightly build ?

**Reply by maicss** [reply]
Ok, thanks.
